src/neo/neo.csproj

```diff
@@ -26,6 +26,7 @@
 
   <ItemGroup>
     <PackageReference Include="Akka" Version="1.4.36" />
+    <PackageReference Include="BLS12381" Version="0.0.2" />
```

Comment:
https://www.nuget.org/packages/Cortex.Cryptography.Bls/

Is this package the same to yours?

Reply:
> https://www.nuget.org/packages/Cortex.Cryptography.Bls/
> 
> Is this package the same to yours?

This one not suport point add,mul and pairing

Reply:
What about https://www.nuget.org/packages/MCL.BLS12_381.Net/

Reply:
> And this one:
> 
> https://www.nuget.org/packages/MCL.BLS12_381.Native/ https://www.nuget.org/packages/MCL.BLS12_381.Net/

This is a similar library to bellman, and is also a wrapper library. I think using our own wrapper would be more beneficial for later cropping.